SummerProject/user/views.py

```diff
@@ -88,7 +88,7 @@ def forgot_password_email_send(request):
             for user in associated_users:
                 content = {
                     'email': user.email,
-                    'domain': request.META['HTTP_HOST'],
+                    'domain': 'localhost:8000',
```

Comment:
Will you please create a constant `HOST = localhost:8000` and use it here instead? Please define this constant in either `settings.py` or `local_settings.py`